cli/internal/cmd/verify.go

```diff
@@ -459,12 +459,12 @@ func (f *attestationDocFormatterImpl) parseSNPReport(b *strings.Builder, reportB
 	writeIndentfln(b, 2, "Platform Info:")
 	writeIndentfln(b, 3, "Symmetric Multithreading enabled (SMT): %t", platformInfo.SMTEnabled)
 	writeIndentfln(b, 3, "Transparent secure memory encryption (TSME): %t", platformInfo.TSMEEnabled)
-	writeIndentfln(b, 2, "Author Key ID: %x", report.AuthorKeyEn)
+	writeIndentfln(b, 2, "Author Key ID: %x", report.SignerInfo)
 	writeIndentfln(b, 2, "Report Data: %x", report.ReportData)
 	writeIndentfln(b, 2, "Measurement: %x", report.Measurement)
 	writeIndentfln(b, 2, "Host Data: %x", report.HostData)
 	writeIndentfln(b, 2, "ID Key Digest: %x", report.IdKeyDigest)
-	writeIndentfln(b, 2, "Author Key Digest: %x", report.AuthorKeyDigest)
+	writeIndentfln(b, 2, "Author Key Digest: %t", ((report.SignerInfo & 1) != 0))
```

Comment:
This change seems unnecessary and wrong. AuthorKeyDigest is still there.

Reply:
Yes, this is accidental